tools/hive-bigquery/bigquery_component.py

```diff
@@ -0,0 +1,632 @@
+"""Module to handle BigQuery related utilities"""
+
+import csv
+import logging
+import os
+import sys
+import time
+
+from google.api_core import exceptions as api_exceptions
+from google.auth import exceptions as auth_exceptions
+from google.cloud import bigquery
+
+import custom_exceptions
+from utilities import get_random_string, print_and_log
+from gcp_service import GCPService
+from properties_reader import PropertiesReader
+
+logger = logging.getLogger('Hive2BigQuery')
+
+
+class BigQueryComponent(GCPService):
+    """BigQuery component to handle functions related to it
```

Comment:
Some of the docstrings were added since pylint was insisting on those, even though they were obvious like in this case and while initializing variables. So Should I proceed and remove those?

Reply:
Make this doc string more descriptive describing what functions this class provides.